addons/mail/security/ir.model.access.csv

```diff
@@ -58,6 +58,8 @@ ir_actions_report_access_user,ir.actions.report.access.user,base.model_ir_action
 access_mail_link_preview_admin,mail.link.preview.admin,model_mail_link_preview,base.group_erp_manager,1,1,1,1
 access_discuss_gif_favorite,discuss.gif.favorite,model_discuss_gif_favorite,base.group_user,1,1,1,1
 access_discuss_voice_metadata_user,discuss.voice.metadata.user,model_discuss_voice_metadata,base.group_system,1,1,1,1
+access_mail_presence,mail.presence,model_mail_presence,base.group_user,1,1,1,1
+access_mail_presence_portal,mail.presence,model_mail_presence,base.group_portal,1,1,1,1

```

Comment:
Does not seem to be linked to any rules ?

Reply:
That's fine too, we can easily sudo when we need.